engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -97,12 +97,14 @@
                                             {% set category_link = display.category.enabled == 'link' %}
                                             <span class="g-array-item-category">
                                                 {% for category in article.categories %}
-                                                    {% if category_link %}
-                                                        <a href="{{ category.route }}">
+                                                    {% if loop.last %}
```

Comment:
Why not `{% for category in article.categories|last %}` instead of looping through all and check for `loop.last`?